<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="sortButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="numberOfElements">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of elements:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="answerBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>471</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="binpowBrowser">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>170</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="element">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="modul">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>60</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Element:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>60</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Modul:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="powerButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>100</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Power</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="timeBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>256</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
